### DISCUSSION TITLE
Plot validation loss

### DISCUSSION THREAD

**Discussion by pedrozamboni** [neuralhydrology/neuralhydrology#126] [General]
Do you have a script to plot train/validaion loss form the log file?

**Comment by gauchm** [comment]
No, but you can nicely look at the loss curves in tensorboard (set `log_tensorboard: True` in the config; start tensorboard via `tensorboard --logdir /path/to/rundir`)
